src/content/bridges/index.md

```diff
@@ -16,7 +16,7 @@ Blockchain bridges work just like the bridges we know in the physical world. Jus
 
 Let's consider an example:
 
-You're from the USA and are planning a trip to Europe. You have USD, but you need EUR to spend. To exchange your USD for EUR you can use a currency exchange for a small fee.
+You're from the USA and are planning a trip to Europe, where only EUR is accepted. You can use a currency exchange for a small fee to exchange your USD for EUR.
```

Comment:
I'm not sure if there is value-add here.. I feel like the simplicity of the three short sentences was intentional here. Example:>Setup.>Problem.>Solution. Would probably leave this

```suggestion
You're from the USA and are planning a trip to Europe. You have USD, but you need EUR to spend. To exchange your USD for EUR you can use a currency exchange for a small fee.
```